### PR TITLE
Added pagination to limit the revisions of a wiki page

### LINKED ISSUE #2699: Displaying only 20 revisions of a page at a time in Revisions tab
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

Some long-running wiki pages have hundreds of revisions, which breaks the browser when you load their revisions tab. Example: https://publiclab.org/openhour has 663 revisions.

The code for this is here:

https://github.com/publiclab/plots2/blob/4b02db50661b9ec519df7712c8f4a915952259e7/app/controllers/wiki_controller.rb#L265-L275

We could use `.page()` to paginate, like this:

https://github.com/publiclab/plots2/blob/4b02db50661b9ec519df7712c8f4a915952259e7/app/controllers/wiki_controller.rb#L333-L334

then add a pagination row to the bottom, like this:

https://github.com/publiclab/plots2/blob/4b02db50661b9ec519df7712c8f4a915952259e7/app/views/wiki/_wikis.html.erb#L17

This is relatively high priority if anyone's available to help out! Thank you!

### PR DESCRIPTION
**Pull request by namangupta01**
Closes #2699 
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: app/controllers/wiki_controller.rb [hunk 1]
```diff
@@ -266,9 +266,10 @@ def revisions
     @node = Node.find_wiki(params[:id])
     if @node
       @revisions = @node.revisions
-      @revisions = @revisions.where(status: 1) unless current_user && current_user.can_moderate?
+      @revisions = @revisions.where(status: 1).page(params[:page]).per_page(20) unless current_user && current_user.can_moderate?
```
Comment (jywarren) [inline app/controllers/wiki_controller.rb]: oh aha - so this `unless` condition turns off the pagination, right? So then we have to be sure not to treat the collection as paginated. You could add the same condition to line 272?

Comment (namangupta01) [reply app/controllers/wiki_controller.rb]: Ohh! 😅  Ya you are right!

**Comment by namangupta01** [comment]
All test passed!

**Comment by jywarren** [comment]
SUPER!!!!!! This is great. @steviepubliclab will be happy - this solves the "zillion revisions" problem.

**Pull request merged**